shared/src/business/entities/EntityConstants.ts

```diff
@@ -1002,6 +1005,7 @@ export const CASE_TYPE_DESCRIPTIONS_WITH_IRS_NOTICE = {
 
 export const CASE_TYPE_DESCRIPTIONS_WITHOUT_IRS_NOTICE = {
   [CASE_TYPES_MAP.cdp]: 'CDP (Lien/Levy)',
+  // [CASE_TYPES_MAP.passport]: 'Passport',
```

Comment:
Can this be removed?

Reply:
Good catch, that should not be commented out